python/lib/dependabot/python/pipenv_runner.rb

```diff
@@ -18,7 +18,7 @@ def run_upgrade(constraint)
         command = "pyenv exec pipenv upgrade #{dependency_name}#{constraint}"
         command << " --dev" if lockfile_section == "develop"
 
-        run(command)
+        run(command, fingerprint: "pyenv exec pipenv upgrade <dependency_name><constraint>")
```

Comment:
🔎 Is the hardcoded `<dependency_name><constraint>` string what's causing these to be aggregated? i.e. Would we get different traces for every combination if we used `#{dependency_name}#{constraint}`?